docs/reference/changelog.md

```diff
@@ -21,40 +22,41 @@
 
 Changed:
 
-* [#261](https://github.com/open-contracting/infrastructure/issues/261) Update description of 'feasibilityStudy' code to include "project"
-* [#267](https://github.com/open-contracting/infrastructure/issues/267) Update description of 'completionCertificate' code to include "project"
+* [#261](https://github.com/open-contracting/infrastructure/issues/261) Update description of 'feasibilityStudy' code to include "project".
+* [#267](https://github.com/open-contracting/infrastructure/issues/267) Update description of 'completionCertificate' code to include "project".
 
 Added:
 
-* [#263](https://github.com/open-contracting/infrastructure/issues/263) 'resettlementPlan'
-* [#265](https://github.com/open-contracting/infrastructure/issues/265) 'financialAgreement'
-* [#266](https://github.com/open-contracting/infrastructure/issues/266) 'budgetAmendmentApproval'
-* [#268](https://github.com/open-contracting/infrastructure/issues/268) 'technicalAuditReport'
-* [#269](https://github.com/open-contracting/infrastructure/issues/269) 'financialAuditReport'
-* [#272](https://github.com/open-contracting/infrastructure/issues/272) 'qualityAssuranceReport'
-* [#274](https://github.com/open-contracting/infrastructure/issues/274) 'incorporationCertificate'
-* [#275](https://github.com/open-contracting/infrastructure/issues/275) 'contractAmendment'
+* [#263](https://github.com/open-contracting/infrastructure/issues/263) 'resettlementPlan'.
+* [#265](https://github.com/open-contracting/infrastructure/issues/265) 'financialAgreement'.
+* [#266](https://github.com/open-contracting/infrastructure/issues/266) 'budgetAmendmentApproval'.
+* [#268](https://github.com/open-contracting/infrastructure/issues/268) 'technicalAuditReport'.
+* [#269](https://github.com/open-contracting/infrastructure/issues/269) 'financialAuditReport'.
+* [#272](https://github.com/open-contracting/infrastructure/issues/272) 'qualityAssuranceReport'.
+* [#274](https://github.com/open-contracting/infrastructure/issues/274) 'incorporationCertificate'.
+* [#275](https://github.com/open-contracting/infrastructure/issues/275) 'contractAmendment'.
+* [#273](https://github.com/open-contracting/infrastructure/issues/273) 'paymentCertificate'.
 
 Removed:
 
-* [#268](https://github.com/open-contracting/infrastructure/issues/268) 'finalAudit'
+* [#268](https://github.com/open-contracting/infrastructure/issues/268) 'finalAudit'.
```

Comment:
For these simple changelog entries, I don't think it makes sense to add a period after the code.

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/288/commits/54bc6f2e541f81e412c5aeee58df7798d921e346